### ISSUE TITLE
Chart first point selected with border around it on iOS

### ISSUE THREAD

**Issue by alokdesai-cuelogic** [highcharts/highcharts-ios#341]
I am using high chart pod version 8.2.1. I am facing weird issue while selecting the points on chart, like when I click on third point bar chart it selects the first bar and draws a rectangular border around it and the selection stays there on first bar even if I click on other bar. This issue is happening with pie and line chart too.
This issue comes on iOS version 13.3 and less.

![bar_chart_issue](https://user-images.githubusercontent.com/16426023/93877343-f1ac2800-fcf5-11ea-9ae4-1523bed2b204.png)

![pie_chart_issue](https://user-images.githubusercontent.com/16426023/93877338-efe26480-fcf5-11ea-918d-71ae8e98df8a.png)

**Comment by ihnatmoisieiev** [comment]
Hello @alokdesai-cuelogic,

thanks for the reporting. Do you have the simplest demo to reproduce the issue?

**Comment by alokdesai-cuelogic** [comment]
Hello,

I have attached the video of your example HighFit application, this video contains the steps to reproduce the issue.
I have run the application on iPad Pro 12.9 and iOS version is 13.3.

[higchart_issue_demo.mov.zip](https://github.com/highcharts/highcharts-ios/files/5274313/higchart_issue_demo.mov.zip)

**Comment by ihnatmoisieiev** [comment]
@alokdesai-cuelogic we have reported the issue here: https://github.com/highcharts/highcharts/issues/14278

You can use the following workaround at the moment:

```
let accessibility = HIAccessibility()
accessibility.keyboardNavigation = HIKeyboardNavigation()
accessibility.keyboardNavigation.enabled = false
options.accessibility = accessibility
```

**Comment by alokdesai-cuelogic** [comment]
I tried the workaround and it is working. Any tentative date when you are releasing this fix.

**Comment by ihnatmoisieiev** [comment]
@alokdesai-cuelogic You can check the status in the linked issue, in my previous comment.

**Comment by alokdesai-cuelogic** [comment]
Hello,

I am still facing the same issue in latest version released v8.2.2. 
Please can you check this on priority.

**Comment by MikolajMichalczak** [comment]
This behavior can be prevented by following these steps::
1. When using the Accessibility module, disable `keyboardNavigation`:
```swift
let accessibility = HIAccessibility()
let kn = HIKeyboardNavigation()
let fb = HIFocusBorder()
kn.focusBorder = fb
accessibility.keyboardNavigation = kn
accessibility.keyboardNavigation.enabled = false
options.accessibility = accessibility
```
   Alternatively, you can disable `FocusBorder`:
```swift
accessibility.keyboardNavigation.focusBorder.enabled = false
```
2. If you do not want to use the Accessibility module, you can also prevent this behavior by disabling it entirely:
```swift
let accessibility = HIAccessibility()
accessibility.enabled = false
```
